src/server/page_server.hpp

```diff
@@ -130,15 +130,15 @@ class page_server
 
       private:
 	// Request handlers for the request server:
-	void receive_boot_info_request (tran_server_conn_t::sequenced_payload &a_ip);
-	void receive_log_page_fetch (tran_server_conn_t::sequenced_payload &a_ip);
-	void receive_data_page_fetch (tran_server_conn_t::sequenced_payload &a_ip);
-	void receive_disconnect_request (tran_server_conn_t::sequenced_payload &a_ip);
-	void receive_log_prior_list (tran_server_conn_t::sequenced_payload &a_ip);
-	void handle_oldest_active_mvccid_request (tran_server_conn_t::sequenced_payload &a_sp);
-	void receive_log_boot_info_fetch (tran_server_conn_t::sequenced_payload &a_ip);
-	void receive_stop_log_prior_dispatch (tran_server_conn_t::sequenced_payload &a_sp);
-	void receive_oldest_active_mvccid (tran_server_conn_t::sequenced_payload &a_sp);
+	void receive_boot_info_request (tran_server_conn_t::sequenced_payload &&a_ip);
+	void receive_log_page_fetch (tran_server_conn_t::sequenced_payload &&a_ip);
+	void receive_data_page_fetch (tran_server_conn_t::sequenced_payload &&a_ip);
+	void receive_disconnect_request (tran_server_conn_t::sequenced_payload &&a_ip);
+	void receive_log_prior_list (tran_server_conn_t::sequenced_payload &&a_ip);
+	void handle_oldest_active_mvccid_request (tran_server_conn_t::sequenced_payload &&a_sp);
+	void receive_log_boot_info_fetch (tran_server_conn_t::sequenced_payload &&a_ip);
+	void receive_stop_log_prior_dispatch (tran_server_conn_t::sequenced_payload &&a_sp);
+	void receive_oldest_active_mvccid (tran_server_conn_t::sequenced_payload &&a_sp);
```

Comment:
For code consistency, since you're revising:
```suggestion
	void receive_boot_info_request (tran_server_conn_t::sequenced_payload &&a_sp);
	void receive_log_page_fetch (tran_server_conn_t::sequenced_payload &&a_sp);
	void receive_data_page_fetch (tran_server_conn_t::sequenced_payload &&a_sp);
	void receive_disconnect_request (tran_server_conn_t::sequenced_payload &&a_sp);
	void receive_log_prior_list (tran_server_conn_t::sequenced_payload &&a_sp);
	void handle_oldest_active_mvccid_request (tran_server_conn_t::sequenced_payload &&a_sp);
	void receive_log_boot_info_fetch (tran_server_conn_t::sequenced_payload &&a_sp);
	void receive_stop_log_prior_dispatch (tran_server_conn_t::sequenced_payload &&a_sp);
	void receive_oldest_active_mvccid (tran_server_conn_t::sequenced_payload &&a_sp);
```